common/xbps-src/shutils/show.sh

```diff
@@ -129,15 +129,16 @@ show_pkg_makedepends() {
 }
 
 show_pkg_build_options() {
-    local f opt desc
+    local f
 
     [ -z "$PKG_BUILD_OPTIONS" ] && return 0
 
     source $XBPS_COMMONDIR/options.description
     msg_normal "$pkgver: the following build options are set:\n"
     for f in ${PKG_BUILD_OPTIONS}; do
-        opt="${f#\~}"
-        eval desc="\${desc_option_${opt}}"
+        local opt="${f#\~}"
+        local descref="desc_option_${opt}"
+        local desc="${!descref-Enable support for $opt}"
```

Comment:
is `${foo-default}` different from `${foo:-default}`?

Reply:
> Omitting the colon results in a test only for a parameter that is unset.

I guess I can change it to `:-` setting an empty description doesn't make much sense.

Reply:
ok just checking, I'm more used to posix parameter expansion